shared/actions/chat/attachment.js

```diff
@@ -25,7 +25,10 @@ function * onShareAttachment ({payload: {message}}: Constants.ShareAttachment):
 function * onSaveAttachmentNative ({payload: {message}}: Constants.SaveAttachment): SagaGenerator<any, any> {
   const {filename, messageID, conversationIDKey} = message
   if (filename && messageID) {
-    const path = downloadFilePath(filename)
+    let path = tmpFile(filename)
```

Comment:
My thinking was that it's being put in the camera roll anyway, so the intermediate move to downloads seemed unnecessary